src/suites/cts/textures/zero_init.spec.ts

```diff
@@ -0,0 +1,1244 @@
+              attachment: renderTexture.createView(),
+              resolveTarget,
+              loadValue: [0, 0, 0, 0],
+              storeOp: C.StoreOp.Store,
+            }],
+            depthStencilAttachment: {
+              attachment: texture.createView(viewDescriptor),
+              depthStoreOp: C.StoreOp.Store,
+              depthLoadValue: C.LoadOp.Load,
+              stencilStoreOp: C.StoreOp.Store,
+              stencilLoadValue: C.LoadOp.Load,
+            },
+          });
+
+          switch (this.params.readMethod) {
+            case ReadMethod.DepthTest:
+              pass.setPipeline(this.getDepthTestReadbackPipeline(state, this.params.format, this.params.sampleCount));
+              break;
+            case ReadMethod.StencilTest:
+              pass.setPipeline(this.getStencilTestReadbackPipeline(this.params.format, this.params.sampleCount));
+              pass.setStencilReference(initializedStateAsStencil(state));
+              break;
+            default:
+              assert(false);
+          }
+          pass.draw(3, 1, 0, 0);
+          pass.endPass();
+
+          const bytesPerPixel = kTextureFormatInfo[C.TextureFormat.R8Unorm].bytes
+          const minBytesPerRow = width * bytesPerPixel;
+          const bytesPerRow = Math.ceil(minBytesPerRow / 256) * 256;
+
+          const byteLength = bytesPerRow * height;
+          const buffer = this.device.createBuffer({
+            size: byteLength,
+            usage: C.BufferUsage.CopySrc | C.BufferUsage.CopyDst,
+          });
+
+          commandEncoder.copyTextureToBuffer(
+            { texture: resolveTexture || renderTexture },
+            {
+              buffer, bytesPerRow, rowsPerImage: height,
+              // @ts-ignore
+              rowPitch: bytesPerRow, imageHeight: height,
+            },
+            { width, height, depth: 1 }
+          );
+
+          this.device.defaultQueue.submit([commandEncoder.finish()]);
+
+          const arrayBuffer = new ArrayBuffer(byteLength);
+          for (let row = 0; row < height; ++row) {
+            new Uint8Array(arrayBuffer, row * bytesPerRow, minBytesPerRow).fill(255);
+          }
+          this.expectContents(buffer, new Uint8Array(arrayBuffer));
+        }
+      } break;
+
+      default:
+        assert(false);
+    }
+  }
+
+  initializeTexture(texture: GPUTexture,
+                    state: InitializedState,
+                    subresourceRange: SubresourceRange) {
+    if (this.params.sampleCount > 1) {
+      // Copies to multisampled textures unspecified.
+      // Use a storeOp for now.
+      assert(kTextureFormatInfo[this.params.format].renderable);
+      this.initializeWithStoreOp(state, texture, this.params.format, this.params.aspect, subresourceRange);
+      return;
+    }
+
+    switch (this.params.dimension) {
+      case '1d':
+      case '3d':
+        // TODO: https://github.com/gpuweb/gpuweb/issues/69
+        // Copies with 1D and 3D textures are unspecified
```

Comment:
```suggestion
        // Copies with 1D and 3D textures are not yet specified
```
(although we'll have them soon)